data/blocklist.json

```
@@ -56,6 +56,7 @@
         "Saras-Daton/amazon_ads",
         "Saras-Daton/amazon_sellerpartner",
         "Saras-Daton/amazon_vendorcentral",
-        "TasmanAnalytics/tasman_dbt_package"
+        "TasmanAnalytics/tasman_dbt_package",
+        "calogica/dbt-date"
```

I'm _guessing_ this needs to be underscore rather than hyphen:

```suggestion
        "calogica/dbt_date"
```